code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm

```diff
@@ -111,15 +111,17 @@
 			qdel(bait)
 			visible_message("<span class='notice'>[src] examines [bait] closer, and telekinetically shatters the pen.</span>")
 
-/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/random/Initialize()
+/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/Initialize() //skyrat edit - every watcher has a chance to be a shiny
 	. = ..()
-	if(prob(1))
-		if(prob(75))
+	if(prob(10)) //skyrat edit
```

Comment:
why is this 10 when goliath is 5.

Reply:
design choice, thought it'd make sense for these to be less rare than the ancient goliath

Reply:
i have conceded